src/DrTests-CommentsToTests-Tests/CommentsToTestsTest.class.st

```diff
@@ -28,7 +28,7 @@ CommentsToTestsTest >> testCommentWithSyntaxError [
 	docComment := thisContext method ast pharoDocCommentNodes first.
 	commentTestCase := CommentTestCase for: docComment.
 	
-	self should: [commentTestCase testIt] raise: SyntaxErrorNotification
+	self should: [commentTestCase testIt] raise: RuntimeSyntaxError
```

Comment:
This difference is from the previous reformatting logic of ast methods. There were two stages:
1. initial parsing of source code in faulty mode
2. reformatting and parsing again with normal mode which causes syntax notification.
Now the second step is removed and therefore only runtime syntax error can be thrown.

Reply:
I always wanted to change the doc comments to store the AST, not the source, but that is for another PR of course